doc/languages-frameworks/python.section.md

```diff
@@ -4,16 +4,7 @@
 
 ### Interpreters {#interpreters}
 
-| Package    | Aliases         | Interpreter |
-|------------|-----------------|-------------|
-| python27   | python2, python | CPython 2.7 |
-| python39   |                 | CPython 3.9 |
-| python310  |                 | CPython 3.10 |
-| python311  | python3         | CPython 3.11 |
-| python312  |                 | CPython 3.12 |
-| python313  |                 | CPython 3.13 |
-| pypy27     | pypy2, pypy     | PyPy2.7 |
-| pypy39     | pypy3           | PyPy 3.9 |
+@python-interpreter-table@
```

Comment:
The output is practically identically. I don't think anyone cares about the ordering of the aliases as long as they are stable.

![image](https://github.com/NixOS/nixpkgs/assets/7258858/b60e7620-1989-4a36-9dab-22ea318e730f)